src/discord-cluster-manager/utils.py

```diff
@@ -58,6 +59,18 @@ async def get_user_from_id(id, interaction, bot):
             return id
 
 
+def discord_followup_wrapper(interaction: discord.Interaction, msg: str) -> None:
+    """
+    To get around response messages in slash commands that are
+    called externally, send a message using the followup.
+    """
+    try:
+        await interaction.response.send_message(msg)
+
+    except Exception:
+        await interaction.followup.send(msg)
```

Comment:
I wonder if this would be better written as the following?
```suggestion
    if not interaction.response.is_done():
        await interaction.followup.send(msg)
    else:
        await interaction.response.send_message(msg)
```
We do call `interaction.response.defer()` when we intend to use followup. And that call to `defer()` ends up making `is_done()` return True.

What makes me uncomfortable about `except Exception` is that we don't really know what's gone wrong.

Honestly, I think it would work either way, but I think I prefer my suggestion if it works.

Reply:
I've changed this, I think this makes sense. Will verify that this works locally before I push though.

Reply:
I also changed the logic for use_followup=True in the followup commit.